docs/tutorials/smart-rollup/set-up.md

```diff
@@ -65,7 +65,7 @@ Later, you will use this image to run a sandbox Tezos environment for testing th
 1. Run this command to start the Docker image, open a command-line terminal in that image, and mount the `hello-world-kernel` folder in it:
 
    ```bash
-   docker run -it --rm --volume $(pwd):/home/tezos/hello-world-kernel --entrypoint /bin/sh --name octez-container tezos/tezos:master
+   docker run -it --rm --volume .:/home/tezos/hello-world-kernel --entrypoint /bin/sh --name octez-container tezos/tezos:master
```

Comment:
I don't know about this. `.` is a relative path where `${pwd}` gives the absolute path, which is prefered in general.

Reply:
Changed this back to `pwd`.